python/tvm/contrib/relay_viz/__init__.py

```diff
@@ -0,0 +1,161 @@
+        if isinstance(node, relay.Function):
+            plotter.node(node_id, "Func", "")
+            plotter.edge(node_to_id[node.body], node_id)
+        elif isinstance(node, relay.Var):
+            name_hint = node.name_hint
+            node_detail = ""
+            node_type = "Var(Param)" if name_hint in relay_param else "Var(Input)"
+            if node.type_annotation is not None:
+                if hasattr(node.type_annotation, "shape"):
+                    shape = tuple(map(int, node.type_annotation.shape))
+                    dtype = node.type_annotation.dtype
+                    node_detail = "name_hint: {}\nshape: {}\ndtype: {}".format(
+                        name_hint, shape, dtype
+                    )
+                else:
+                    node_detail = str(node.type_annotation)
+            plotter.node(node_id, node_type, node_detail)
+        elif isinstance(node, relay.Tuple):
+            plotter.node(node_id, "Tuple", "")
+            for field in node.fields:
+                plotter.edge(node_to_id[field], node_id)
+        elif isinstance(node, relay.expr.Constant):
+            node_detail = "shape: {}, dtype: {}".format(node.data.shape, node.data.dtype)
+            plotter.node(node_id, "Const", node_detail)
+        elif isinstance(node, relay.expr.Call):
+            op_name = unknown_type
+            node_details = []
+            if isinstance(node.op, tvm.ir.Op):
+                op_name = node.op.name
+                if node.attrs:
+                    node_details = [
+                        "{}: {}".format(k, node.attrs.get_str(k)) for k in node.attrs.keys()
+                    ]
+            elif isinstance(node.op, relay.Function):
+                func_attrs = node.op.attrs
+                op_name = "Anonymous Func"
+                if func_attrs:
+                    node_details = [
+                        "{}: {}".format(k, func_attrs.get_str(k)) for k in func_attrs.keys()
+                    ]
+                    if "Composite" in func_attrs.keys():
+                        op_name = func_attrs["Composite"]
+            else:
+                op_name = str(type(node.op)).split(".")[-1].split("'")[0]
+
+            plotter.node(node_id, op_name, "\n".join(node_details))
+            args = [node_to_id[arg] for arg in node.args]
+            for arg in args:
+                plotter.edge(arg, node_id)
+        elif isinstance(node, relay.expr.TupleGetItem):
+            plotter.node(node_id, "TupleGetItem", "idx: {}".format(node.index))
+            plotter.edge(node_to_id[node.tuple_value], node_id)
+        elif isinstance(node, tvm.ir.Op):
+            pass
+        elif isinstance(node, relay.Let):
+            plotter.node(node_id, "Let", "")
+            plotter.edge(node_to_id[node.value], node_id)
+            plotter.edge(node_id, node_to_id[node.var])
+        else:
+            unknown_info = "Unknown node: {}".format(type(node))
```

Comment:
One issue I found when partitioning `mobilenet_v1_1.0_224_quant.tflite` with ethos-n before building it, I got:
```
Unknown node: <class 'tvm.ir.expr.GlobalVar'>
```

Reply:
Ah, Thanks! will handle global var.